include/dgl/graph_op.h

```diff
@@ -139,6 +139,13 @@ class GraphOp {
    * \return a new immutable bidirected graph.
    */
   static GraphPtr ToBidirectedImmutableGraph(GraphPtr graph);
+  /*!
+   * \brief Same as BidirectedMutableGraph except that the returned graph is immutable
+   * and call gk_csr_MakeSymmetric in GKlib. It works much faster.
```

Comment:
.... It utilizes a more efficient implementation using Gklib. Return a null pointer if the conversion fails.